README.md

```diff
@@ -362,34 +362,41 @@ See also [`addTimer()`](#addtimer) for more details.
 
 An `ext-event` based event loop.
 
-This uses the [`event` PECL extension](https://pecl.php.net/package/event).
-It supports the same backends as libevent.
+This uses the [`event` PECL extension](https://pecl.php.net/package/event),
+that provides an interface to `libevent` library.
+`libevent` itself supports a number of system-specific backends (epoll, kqueue).
 
 This loop is known to work with PHP 5.4 through PHP 7+.
 
 #### ExtEvLoop
 
 An `ext-ev` based event loop.
 
-This loop uses the [`ev` PECL extension](https://pecl.php.net/package/ev), that
-provides an interface to `libev` library.
+This loop uses the [`ev` PECL extension](https://pecl.php.net/package/ev),
+that provides an interface to `libev` library.
+`libev` itself supports a number of system-specific backends (epoll, kqueue).
+
 
 This loop is known to work with PHP 5.4 through PHP 7+.
 
 #### ExtUvLoop
 
 An `ext-uv` based event loop.
 
-This loop uses the [`uv` PECL extension](https://pecl.php.net/package/uv), that
-provides an interface to `libuv` library.
+This loop uses the [`uv` PECL extension](https://pecl.php.net/package/uv),
+that provides an interface to `libuv` library.
+`libuv` itself supports a number of system-specific backends (epoll, kqueue).
 
 This loop is known to work with PHP 7+.
 
-#### ExtLibeventLoop
+#### ~~ExtLibeventLoop~~
```

Comment:
LGTM, but make sure to also update the TOC similar to #231 :+1:

Reply:
Done 👍